src/screens/ProductDetails/ProductDetails.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import Screen from '../../styledComponents/Screen'
+import Button from '../../styledComponents/Button'
+import products from '../../data/products'
+import categories from '../../data/categories'
+import {connect} from 'react-redux'
+import { Link } from 'react-router-dom'
+import {addToCart} from '../../screens/ShoppingCart/actionCreators'
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    addToCart: (product,amount) => dispatch(addToCart(product,amount))
```

Comment:
It seems, that amount isn't really supported, right?
If, for some reasons, it has to be here, it would at least set it with default value =1.